pyproject.toml

```
@@ -14,14 +14,24 @@ license = { file = "licenses/LICENSE.rst" }
 authors = [
     { name = "The Astropy Developers", email = "astropy.team@gmail.com" }
 ]
-dependencies = []
+dependencies = [
+  "array-api-compat",
+  "astropy",
```

```suggestion
  "astropy>=7.0.0",
```

Or an earlier version.